<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceChatWindow</class>
 <widget class="QWidget" name="VoiceChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>didactic-pancake</string>
  </property>
  <widget class="QLabel" name="lbl_avatar">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>159</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_hang_up">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_target_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>334</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wait_ball_1">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>404</y>
     <width>6</width>
     <height>6</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wait_ball_2">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>404</y>
     <width>6</width>
     <height>6</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wait_ball_3">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>404</y>
     <width>6</width>
     <height>6</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_text1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>300</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>正在等待对方接受邀请</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>561</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hang_up">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>561</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>挂断</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_accept">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reject">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_reject">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>挂断</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_accept">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>接听</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_chat_time">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>2</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
